src/Tribe/Attendees.php

```diff
@@ -751,73 +753,79 @@ public function sanitize_csv_value( $value ) {
 	 *
 	 * @since 4.6.2
 	 *
+	 * @param bool $all Whether to generate a CSV for attendees of all events or just the current one.
+	 *
+	 * @return void
 	 */
-	public function maybe_generate_csv() {
-		if ( empty( $_GET['attendees_csv'] ) || empty( $_GET['attendees_csv_nonce'] ) || empty( $_GET['event_id'] ) ) {
+	public function maybe_generate_csv( $all = false ) {
+		if ( ! isset( $_GET['attendees_csv_nonce'] )
```

Comment:
Minor: For multiline if statements, we should put the first comparison on a new line and leave `if (` on its own line

```suggestion
		if (
			! isset( $_GET['attendees_csv_nonce'] )
```